docs/tutorials/notebooks/basic_ot_between_datasets.ipynb

```diff
@@ -0,0 +1,387 @@
+{
```

Comment:
<p>thanks for writing this extra paragraph, it's very cool!</p><p> </p><p>"of different masses" -&gt; "of different total masses."</p><p> </p><p>I also feel it would be great to have links, either to other tutorials or maybe to the functions, rather than plain text. An extra line on neural approaches could be nice.</p><p> </p><p>BTW, Low-rank Sinkhorn is not so much about using a low-rank approx of the cost matrix (this can be used), but rather implement a low-rank <strong>constraint </strong>on the coupling.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>I have now added these new approaches. Not sure where the neural approaches fit here, I can also put it in the "numerical tricks to make ot scale", wdyt?</p>